lib/cmd/common/import.js

```diff
@@ -122,12 +124,21 @@ function importAppNgui(projectId, appTitle, appTemplateType, fileOrRepo, deployE
       connections:[],
       template: {
         repoUrl: repo,
-        type: appTemplateType
+        type: appTemplateType,
+        imported: true,
+        repoBranch: 'refs/heads/' + repoBranch
       }
     };
 
     var url = 'box/api/projects/' + projectId + '/apps';
-    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload,  i18n._("Error importing git repo: "), cb);
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, payload, i18n._("Error importing git repo: "), function(err, data) {
+      if (err) {
+        return cb(err);
+      }
+
+      log.silly('import response', data);
+      return cb(null, 'Import Successful.');
```

Comment:
@alanmoran I tend to agree that returning big ugly json is not user friendly but returning just `Import Successful` is just too succinct in my opinion. I would expect at least generated cloud app id to be returned, might be something like `Import Successful. To see more details of your cloud app use fhc app read --project=<projectId> --app=<generatedAppId>` where the project and app Ids are replaced with actual values.

Also big ugly json might make test automation a bit simpler.

In general I don't like the inconsistencies here. When importing the cloud app from zip file, the big ugly json is still returned. I would prefer not to touch this at all or fix it for all sources (zip file, git repo, bare git repo) at once.